### ISSUE TITLE
Error message 

### ISSUE THREAD

**Issue by elextr** [geany/geany#1862]
The following occurred, not sure what I was doing to cause it.  This is to record it, don't have time to do the debug just ATM.

```
*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug
```

Slightly olde git build:
```
10:50:32: Geany INFO		: Geany 1.33 (git >= 58aa099), en_AU.UTF-8
10:50:32: Geany INFO		: GTK 3.18.9, GLib 2.48.2
```

**Comment by b4n** [comment]
Did this crash Geany?  I think I saw something like that a few times, and the best I could figure out was a bug in GTK or below.  Not sure though.

**Comment by elextr** [comment]
No, just terminal messages, so I didn't see them when they happened, so I'm not sure what I did.  Do have split window plugin open.

**Comment by elextr** [comment]
Note this is on an old laptop, so the build of Geany is pre-1.33 release, but I'm not aware of anything that addressed stuff in this area for that release.

**Comment by codebrainz** [comment]
That doesn't look like a Geany problem, likely fixed since the older version of pixman you have. I get quite a few GTK+ "critical" spams with GTK+3 build, but I don't think I've seen that one.

**Comment by elextr** [comment]
I normally don't get any messages, so if you are getting "quite a few" better post issues so they can be fixed.

**Comment by codebrainz** [comment]
They aren't caused by Geany, like the one in this PR.

**Comment by shevegen** [comment]
I remember this problem with other applications too. I am not sure if the "fixed in newer pixman releases" is correct, though, since I always recompile from source using the latest and greatest (when possible).

**Comment by elextr** [comment]
And quick as a flash it hasn't happened since.
